services/user.py

```diff
@@ -0,0 +1,44 @@
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                email: str = None,
+                first_name: str = None,
+                last_name: str = None) -> User:
+    user = User.objects.create_user(username=username,
+                                    password=password)
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()
+    return user
+
+
+def get_user(user_id: str) -> User:
+    return User.objects.get(id=user_id)
+
+
+def update_user(user_id: str, username: str = None,
+                password: str = None,
+                email: str = None,
+                first_name: str = None,
+                last_name: str = None) -> None:
```

Comment:
The return type of the `update_user` function is incorrectly specified as `None`. It should be `User` to accurately reflect the type of object being returned.